src/sinks/prometheus/exporter.rs

```diff
@@ -80,6 +80,7 @@ pub struct PrometheusExporterConfig {
     ///
     /// The metrics are exposed at the typical Prometheus exporter path, `/metrics`.
     #[serde(default = "default_address")]
+    #[configurable(metadata(docs::examples = "0.0.0.0:9598"))]
```

Comment:
**nitpick (non-blocking)**: Having the example the same as the default always bothers me